webapp/src/components/news-form/NewsForm.vue

```diff
@@ -31,7 +33,7 @@
 <script>
 export default {
   name: 'NewsForm',
-  'props': ['uniqueTitles'],
+  props: ['uniqueTitles'],
```

Comment:
It's best practice to do:
```suggestion
  props: {
    uniqueTitles: { type: Array, required: true }
  }
```
So you always get a nice warning when you render your component and forgot to set the properties (which you did in your storybook story).